### PR TITLE
Enable Edge to Edge Display

### PR DESCRIPTION
**Pull request by Sheerahjay**
### **User description**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

___

### **PR Type**
Enhancement

___

### **Description**
• Enable edge-to-edge display for modern Android UI
• Update window insets handling in Compose screens
• Fix layout padding and system window compatibility
• Set portrait orientation in AndroidManifest

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>MainActivity.kt</strong><dd><code>Enable edge-to-edge display in MainActivity</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

sample/src/main/java/com/smileidentity/sample/activity/MainActivity.kt

• Import and call <code>enableEdgeToEdge()</code> in <code>onCreate()</code><br> • Enable modern <br>edge-to-edge display functionality

</details>

  </td>
  <td><a href="https://github.com/smileidentity/android/pull/639/files#diff-75bf05d75b81700b4bcf8c2a2a499fe64384ea4159cf7520659fe93417490332">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>MainScreen.kt</strong><dd><code>Update window insets handling in Compose</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

sample/src/main/java/com/smileidentity/sample/compose/MainScreen.kt

• Replace deprecated <code>consumeWindowInsets</code> with <code>WindowInsets.systemBars</code><br> <br>• Update Scaffold to use <code>contentWindowInsets</code> parameter<br> • Simplify <br>padding handling in NavHost content

</details>

  </td>
  <td><a href="https://github.com/smileidentity/android/pull/639/files#diff-ab8457b73dfbd511f5bef81764d13d6ba338f1ea305e9909c44e51f8c4d21011">+5/-5</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>activity_java.xml</strong><dd><code>Add system window fitting to layout</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

sample/src/main/res/layout/activity_java.xml

• Add <code>android:fitsSystemWindows="true"</code> to FrameLayout<br> • Ensure proper <br>system window handling in XML layout

</details>

  </td>
  <td><a href="https://github.com/smileidentity/android/pull/639/files#diff-fb9a47736bc0bc011444b9b46736b6291c44b96596979a590d274b9376b0d565">+2/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Miscellaneous</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>AndroidManifest.xml</strong><dd><code>Clean up AndroidManifest configuration</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

sample/src/main/AndroidManifest.xml

• Remove unused <code>android:label</code> attribute from JavaActivity<br> • Clean up <br>manifest configuration

</details>

  </td>
  <td><a href="https://github.com/smileidentity/android/pull/639/files#diff-c53f7bc57c4a7ffb92824cc2598ff253042b28a0d38f69eead012e62f7c41936">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> <details> <summary>  Need help?</summary><li>Type <code>/help how to ...</code> in the comments thread for any questions about PR-Agent usage.</li><li>Check out the <a href="https://qodo-merge-docs.qodo.ai/usage-guide/">documentation</a> for more information.</li></details>

### CODE REVIEW

**Review by jumaallan — APPROVED** [review]
Thank you @Sheerahjay and @wangerekaharun 👯

File: sample/src/main/java/com/smileidentity/sample/compose/MainScreen.kt [hunk 1]
```diff
@@ -111,7 +112,7 @@ fun MainScreen(
         }
     }
     Scaffold(
-        modifier = modifier,
+        contentWindowInsets = WindowInsets.systemBars,
         snackbarHost = { Snackbar() },
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply sample/src/main/java/com/smileidentity/sample/compose/MainScreen.kt:116]: It's our sample app so we ignore this 😄
